### ISSUE TITLE
Edit Toolbar - Bug with delete button when item title contains the simple quote caracter « ' »

### ISSUE THREAD

**Issue by GwendolinePouet** [2sic/2sxc#1688]
**I'm submitting a ...**  
[x] bug report 

**...about**   
[x] edit experience / UI

**Current behavior**
In Edit Mode (on my 2sxc App where data are managed by Data Query), the toolbar including delete button doesn't appear when the name/title (taken back by EntityTitle) of the item contains quote caracter ( ' ). Problem seems arise from the data JSON parsing.

My toolbar code on the template :
~~~~
@Edit.Toolbar(cont, toolbar: new { 
    action = "edit,delete",
    entityId = cont.EntityId, 
    entityGuid = cont.EntityGuid, 
    entityTitle = cont.EntityTitle 
})
~~~~

My console error :
![image](https://user-images.githubusercontent.com/15940924/53582923-cba5bc80-3b80-11e9-99f9-49e235b368d7.png)

**Expected behavior**
Toolbar with delete button should be appear and work .

**Minimal reproduction of the problem with instructions**
Create a new app (like a news app) where data are managed by the Data Query and not by a Content Type. Create a template with a list and a toolbar with delete button in each item . Create a first content without simple quote caracter in the title and after an other with a quote in the title.

**Please tell us about your environment:**

* **2sxc version(s):** 9.3.5  
* **Browser:** all
* **DNN:** 7.4.2
* **Language:** French

**Comment by iJungleboy** [comment]
Sound bad, we'll see what we can do about it.

**Comment by iJungleboy** [comment]
just fixed it - will be in 9.40
